src/Toolkit/Toolkit/UI/Controls/FeatureForm/FieldFormElementView.cs

```diff
@@ -118,12 +118,12 @@ private async void OnValuePropertyChanged()
             try
             {
                 await FeatureForm.EvaluateExpressionsAsync();
-                var errors = Element?.GetValidationErrors();
+                var errors = Element.GetValidationErrors();
 
                 string? errMessage = null;
                 if (errors != null && errors.Any())
                 {
-                    errMessage = string.Join("\n", errors.Select(e => e.Message));
+                    errMessage = string.Join("\n", errors.Select(e => FeatureFormView.ValidationErrorToLocalizedString(Element, e)!));
```

Comment:
Should this be `Environment.NewLine`? Or does `\n` work everywhere?

Reply:
Works everywhere.